boards/arm/actinius_icarus_som_dk/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2022 Actinius
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+if(CONFIG_BOARD_ACTINIUS_ICARUS_SOM_DK_NS)
+	zephyr_library_sources(board_ns.c)
+else()
+	zephyr_library_sources(board.c)
+endif()
```

Comment:
`board.c` is an empty file, so please delete it and just do

```
if(CONFIG_BOARD_ACTINIUS_ICARUS_SOM_DK_NS)
  zephyr_library()
	zephyr_library_sources(board_ns.c)
endif()
```

Reply:
This whole block is not used after the introduction of the common lib. Should I squash the 2 commits so that the som dk and the common lib come in in a single commit?

Reply:
I think so, yes

Reply:
Done!